src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,41 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const {
+    user,
+    comments,
+    title,
+    body,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        {user && (
+          <p>
+            {' Posted by  '}
+
+            <UserInfo user={user} />
+          </p>
+        )}
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      <hr />
+
+      {comments.length > 0
+        ? <CommentList comments={comments} />
+        : <b data-cy="NoCommentsMessage">No comments yet</b>
+      }
```

Comment:
you can also use comments.length (shorthand), but you have to convert it to boolean, either with !! or with Boolean